packages-next/types/src/config/index.ts

```diff
@@ -85,6 +85,8 @@ export type AdminUIConfig = {
     isValidSession: boolean;
     createContext: CreateContext;
   }) => MaybePromise<{ kind: 'redirect'; to: string } | void>;
+  /** Enables nextjs graphql api route mode */
+  enableGraphQlApiEndpoint?: boolean;
```

Comment:
This will need to be documented in the  API docs in the "Advanced configuration:" section of `docs-next/pages/apis/config.mdx`.

Reply:
oof, is that how we capitalise `enableGraphQlApiEndpoint`?

Took a look at the types and we're not super consistent here, but an equivalent config option is `extendGraphqlSchema` so I'm pretty sure we decided that "Graphql" is just a word in terms of camelcasing

Also I get why this is nested under `ui` but it's ... weird. I misunderstood what this option meant on the first read. We should come back to that later, in the meantime my preferred name for this is

```
  /** Enables nextjs graphql api route mode */
  addNextjsGraphqlEndpoint?: boolean;
```

Basically, tells the ui to "add a Next.js GraphQL Endpoint", and then when we're creating an express server, if that's the case we don't also add an express handler for that.

Reply:
Just saw this comment, okay can easily change this

Reply:
I think enable is better for boolean values though as a key

Reply:
also do we need to document stuff under `experimental` _right now_?